### PR TITLE
sys-apps/systemd: Update to systemd v247

### PR DESCRIPTION
**Pull request by sayanchowdhury**
Signed-off-by: Sayan Chowdhury <sayan@kinvolk.io>

# sys-apps/systemd: Update to systemd v247

The patches are adjusted, as they were failing to apply. 

# How to use
```
emerge-amd64-usr systemd
./build_image
```

# Testing done

Scheduled a Jenkins [build](http://localhost:9091/job/os/job/manifest/1842/cldsv/)

### CODE REVIEW

File: sys-apps/systemd/systemd-9999.ebuild [hunk 1]
```diff
@@ -401,16 +401,16 @@ multilib_src_install_all() {
...
-		rm -r "${ED}${rootprefix}"/lib/udev/hwdb.d || die
-	fi
-
 	# Flatcar: Upstream uses keepdir commands to keep some empty
 	# directories.
 	#
 	# Flatcar: TODO: Consider using that instead of
 	# systemd_dotmpfilesd "${FILESDIR}"/systemd-flatcar.conf below.
 
+	if use hwdb; then
```
Comment (krnowak) [inline sys-apps/systemd/systemd-9999.ebuild:410]: Any reason for this change?

**Review by krnowak — APPROVED** [review]
LGTM. But had one question.

File: sys-apps/systemd/systemd-9999.ebuild (hunk 1 above)
Comment (sayanchowdhury) [reply sys-apps/systemd/systemd-9999.ebuild:410]: I was using `vimdiff` for comparing the diff with upstream, and the `# Flatcar` comment seemed out of place.

Comment (krnowak) [reply sys-apps/systemd/systemd-9999.ebuild:410]: Alright.

I think this comment was about the stuff that exists in upstream ebuild, but we have dropped. So this comment is probably bound to seem out of place.

**Pull request merged**